### ISSUE TITLE
Cloudflare Challenge Detected

### ISSUE THREAD

**Issue by HemHem07** [Spark-NF/firefox-novel-updates#32]
Since a week-ish ago, I've just been getting Cloudflare Challenge detected (and consequently, no reading list showing up). Reverting to prior versions shows just an error.

**Comment by Spark-NF** [comment]
Indeed, I can reproduce on my end as well, weird. Normally clicking "Open" and passing the CloudFlare challenge should suffice, but it seems that's not the case anymore. I'll have a look 👍

**Comment by Spark-NF** [comment]
After some investigation, it would seem that the `cf_clearance` cookie is not being sent in the requests from the extension, even when it exists, causing Cloudflare to block the HTTP call. I'm investigating some more 🤔

**Comment by Spark-NF** [comment]
This is caused by the `cf_clearance` cookie being partitioned ([CHIPS](https://developer.mozilla.org/en-US/docs/Web/Privacy/Privacy_sandbox/Partitioned_cookies)).

A similar issue has been reported here already: https://github.com/privacycg/CHIPS/issues/6#issuecomment-2392660282 I couldn't find anything related on Mozilla bug reports, but I found this for Chrome: https://groups.google.com/a/chromium.org/g/chromium-extensions/c/eE1rMKYfNbI?pli=1.

I'll continue looking for alternatives, since the cookie is technically available through the cookie API, just not sent via `fetch` / `XMLHttpRequest`.

**Comment by Spark-NF** [comment]
Alright, I found a solution by creating a copy of the cookie without the `partitionKey` using the `browser.cookies` API. The current `master` branch doesn't work without Vue 3 so I'll release a new version once https://github.com/Spark-NF/novel-updates-notifier/issues/35 is ironed out.
